### ISSUE TITLE
Create offer -Product category is required. Enter the category ID.

### ISSUE THREAD

**Issue by xu19950827** [allegro/allegro-api#7309]
![DC7F36UM95B{{5M3Q2XRGY1](https://github.com/allegro/allegro-api/assets/22289367/954d1c41-bb10-49a3-8449-5200515d8f5e)

**Comment by MaciejFrackowiak** [comment]
If you try to list an offer with a product, which doesn't exist in our product base you have to provide all the necessary details for the product. One of them is a category which should also be provided in the product part of the request - please, check [our tutorial](https://developer.allegro.pl/tutorials/list-offer-assigned-product-one-request-D7Kj9M71Bu6#how-to-list-an-offer-assigned-to-a-new-product).

**Comment by xu19950827** [comment]
Is it added in the productSet field? Example:
![image](https://github.com/allegro/allegro-api/assets/22289367/180325bd-2c3d-415a-a1d4-6945a47484a6)

**Comment by MaciejFrackowiak** [comment]
Yes, it should be added in the productSet structure as you show it on the image.
